### DISCUSSION TITLE
Can you use the app with *.cloudflareaccess.com?

### DISCUSSION THREAD

**Discussion by ShrimpyChips** [advplyr/audiobookshelf-app#1313] [Q&A] [answered]
I recently moved my home server onto Cloudflare and I have it setup to authenticate via my google account. So the first time I go to the server it will redirect me to myservername.cloudflareaccess.com to handle the authentication and then hand back to my server URL. This all works fine in the browser but in the app I get "Server redirected somewhere else (to myservername.cloudflareaccess.com).

Sorry I'm new to the whole Cloudflare setup but is there a way to get this to work?

**Comment by nichwall** [comment, marked as answer]
Sounds like you're trying to use forward authentication. Forward authentication is not supported by the apps and will only work with the web browser.

Relevant discussion here https://github.com/advplyr/audiobookshelf-app/issues/1155#issuecomment-2031674669

**Comment by ShrimpyChips** [comment]
Thanks, that was kind of what I suspected. I ended up setting up Tailscale to resolve the problem.

**Comment by StupydHors** [comment]
I would really like this to work, but I'm not confident in my app dev abilities. I can use Zerotier/Tailscale or another VPN but its kinda annoying to always have to turn it on.
